src/main/java/core/basesyntax/data/servise/FruitShopService.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.data.servise;
+
+import core.basesyntax.data.exeption.UnknownOperationException;
+import core.basesyntax.data.model.FruitTransaction;
+import core.basesyntax.data.model.OperationType;
+import core.basesyntax.data.strategy.BalanceOperationHandler;
+import core.basesyntax.data.strategy.OperationHandler;
+import core.basesyntax.data.strategy.PurchaseOperationHandler;
+import core.basesyntax.data.strategy.ReturnOperationHandler;
+import core.basesyntax.data.strategy.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopService {
+    private Map<String, Integer> fruitInventory = new HashMap<>();
+    private Map<OperationType, OperationHandler> operationStrategy = new HashMap<>();
```

Comment:
Consider making `fruitInventory` and `operationStrategy` private and final to adhere to encapsulation principles and ensure these maps are not modified outside of this class.